runelite-client/src/main/resources/net/runelite/client/plugins/skybox/skybox.txt

```diff
@@ -957,7 +957,14 @@ c 7 4
 
 m31 89
 
-// 31 89 p0 Hosidius
+// 31 89 p1 Christmas-themed POH
+p 1
+#FFFFFF
```

Comment:
Pure white is a bit too bright [based on your screenshot](https://github.com/runelite/runelite/issues/14120#issuecomment-917861144). Maybe tone this down to something like `#BBB`? Play with the color and get it closer to the color of the snow.

Reply:
the color should be `#b6cbd8`

Reply:
@abextm I just did so now in a PR.